client/proto/request/v1/request.proto

```diff
@@ -34,7 +35,10 @@ service RequestService {
 
 message Request {
   // Required application request which gets evaluated by the execution engine.
-  bytes request = 1;
+  oneof application_request {
+    bytes raw_request = 1 [deprecated = false];
```

Comment:
If you don't want to deprecate the previous field yet, then please remove the flag as it's `false` by default. See the effects of this flag [here](https://developers.google.com/protocol-buffers/docs/proto3#options), i.e., it just adds an annotation to generated classes.
```suggestion
    bytes raw_request = 1;
```

Reply:
Updated changes